native/cocos/renderer/gfx-gles3/GLES3GPUObjects.h

```diff
@@ -156,6 +156,8 @@ struct GLES3GPUTextureView {
     Format format = Format::UNKNOWN;
     uint32_t baseLevel = 0U;
     uint32_t levelCount = 1U;
+    uint32_t basePlane = 0U;
+    uint32_t planeCount = 0U;
```

Comment:
We may need a texture subrange struct to avoid missing subranges usage scenarios, such as the texture barrier. However, it may break compatibility.

Reply:
sure, subrange is upcoming